lib/widgets/features/inbox/screens/inbox_invitation_detail.dart

```diff
@@ -405,21 +463,25 @@ class _InboxInvitationDetailScreenState
 
 class DeclineReason extends StatefulWidget {
   final String? name;
+  int selectedReason;
```

Comment:
@saroj-kumar-padhi I'm no Flutter developer, but as I understand it you `DeclineReason` should be an immutable object and you should not add a state variable to it. Any user input should be saved into `_DeclineReasonState`.